app/controllers/spotlight/catalog_controller.rb

```diff
@@ -51,7 +51,10 @@ def autocomplete
   def admin
     self.blacklight_config.view.select! { |k,v| k == :admin_table }
     self.blacklight_config.view.admin_table.partials = [:index_compact]
-    self.blacklight_config.sort_fields.reject! { |k,v| true }
+
+    unless self.blacklight_config.sort_fields.has_key? :timestamp
+      self.blacklight_config.add_sort_field :timestamp, sort: "#{blacklight_config.index.timestamp_field} desc"
```

Comment:
Yeah, maybe not passing the search context would be easiest. I'll see how possible that is.